src/orangepi/ProductionFiles/FinalPiProgram.py

```diff
@@ -231,8 +398,6 @@ def average_position_of_pixels(mat, threshold=128):
         output = denoise_image(output)
```

Comment:
what is output? where is that coming from???  is this supposed to be denoise_image(output)???